runtime-extension-model/src/main/resources/META-INF/mule-operation.xsd

```diff
@@ -0,0 +1,268 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
did you code this manually of is this generated?

Reply:
Manually written. Downsides on making it auto generated